### ISSUE TITLE
Remove tgz files from source package

### ISSUE THREAD

**Issue by haibow** [apache/incubator-pinot#5540]
It was flagged by @felixcheung that there are binary files in the source package
```
./kubernetes/helm/pinot-0.2.0.tgz
./kubernetes/helm/pinot/charts/zookeeper-2.1.3.tgz
./kubernetes/helm/presto-0.2.0.tgz
```
Creating this issue for tracking.
Options:
- Use raw files
- Exclude certain folders from src release

**Comment by fx19880617** [comment]
We should exclude:
```
./docker
./kubernetes
./website
```

**Comment by snleee** [comment]
Addressed in the #5541 please take a look
